PyISY/Nodes/__init__.py

```diff
@@ -187,8 +187,22 @@ def parse(self, xml):
                                 dimmable = '%' in \
                                     nprop[0].attributes['uom'].value
                                 nval = int(nval.replace(' ', '0'))
+                            # Get the device notes, currently only the Spoken property is saved.
+                            notes_xml = self.parent.conn.getNodeNotes(nid)
```

Comment:
Pretty much. I think that would be a better option because this data is not sent in the bulk XML data and it is somewhat ancillary.